CHANGELOG.md

```diff
@@ -10,8 +10,10 @@
 
 ### Added
 
-- Added command that formats/prettify all `.tf`-files in directory #21
+- ~~Added command that formats/prettify all `.tf`-files in directory #21~~
```

Comment:
Why strikethrough?

Reply:
Because I removed that command and put it under `make pre-commit`.

Reply:
Aha, I see. Maybe add a `(overwritten by #24)` or something instead of strikethrough. A little more explicit

Reply:
Yeah, that's a good idea. i'll do that.